Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.py

```diff
@@ -0,0 +1,244 @@
+        else:
+            raise e
+    return message
+
+
+def handle_errors(result: Dict):
+    """
+    Handle errors, raise Exception when there is errors in the response.
+    """
+
+    status_code = result.get('StatusCode', 0)
+    result_body = result.get('Body')
+
+    if result_body == '' and status_code == 204:
+        raise Exception('No content received. Possible API rate limit reached.')
+
+    if 200 < status_code < 299:
+        raise Exception(f'Failed to perform request, request status code: {status_code}.')
+
+    if result_body == '':
+        raise Exception('No content received. Maybe you tried a private API?.')
+
+    if result.get('error'):
+        error_massage = result.get('error', {}).get('message')
+        error_code = result.get('error', {}).get('code')
+        raise Exception(f'Failed accessing Google Safe Browsing APIs. Error: {error_massage}. Error code: {error_code}')
+
+
+def arrange_results_to_urls(results: List, url_list: List) -> Dict:
+    """ arrange the results according the urls list.
+    Args:
+        results: the response.
+        url_list: The urls list
+    Returns:
+        (dict) The results according the urls.
+    """
+    urls_results: Dict[str, list] = {}
+    for url in url_list:
+        urls_results[url] = []
+
+    for result in results:
+        url = result.get('threat', {}).get('url')
+        urls_results[url].append(result)
+
+    return urls_results
+
+
+def url_command(client: Client, args: Dict[str, Any], client_body) -> CommandResults:
+    """
+    url command: Returns URL details for a list of URL
+    """
+
+    url = argToList(args.get('url'))
+
+    result = client.url_request(client_body, url)
+
+    if result.get('StatusCode'):
+        handle_errors(result)
+
+    if not result:
+        handle_errors(result)
+
+    urls_data = arrange_results_to_urls(result.get('matches'), url)
+
+    url_data_list = []
+    for url_key, url_data in urls_data.items():
+        if url_data:
+            dbot_score = Common.DBotScore(
+                indicator=url_key,
+                indicator_type=DBotScoreType.URL,
+                integration_name='GoogleSafeBrowsingV2',
+                score=3,
+                reliability=client.reliability
+            )
+            url_standard_context = Common.URL(
+                url=url_key,
+                dbot_score=dbot_score
+            )
+            url_data_list.append(CommandResults(
+                readable_output=tableToMarkdown(f'Google Safe Browsing APIs - URL Query: {url_key}', url_data),
+                outputs_prefix='GoogleSafeBrowsingV2.URL',
+                outputs_key_field='IndicatorValue',
+                outputs=url_data,
+                indicator=url_standard_context
+            ))
+        else:
+            dbot_score = Common.DBotScore(
+                indicator=url_key,
+                indicator_type=DBotScoreType.URL,
+                integration_name='GoogleSafeBrowsingV2',
```

Comment:
put it as a const/global.
also - use `GoogleSafeBrowsing` and not v2, to not break BC.

Reply:
done